manager/src/main/java/com/redhat/service/smartevents/manager/workers/resources/ConnectorWorker.java

```diff
@@ -153,6 +169,7 @@ private ConnectorEntity deployConnector(ConnectorEntity connectorEntity) {
     }
 
     @Override
+    @ActivateRequestContext
```

Comment:
@manstis Could you add a comment here just to explain why the `@ActivateRequestContext` is required (and should not be removed).

cheers,
Rob